jenkins/jjb/templates/scenario-tests.yaml

```diff
@@ -146,4 +146,172 @@
+      - string:
+          name: BUILD_NUM
+          default: ''
+          description: 'Build number to test. Get latest build in the google storage bucket folder if not specified'
+      - string:
+          name: NIMBUS_USER
+          default: ''
+          description: 'Must specified. The username of the nimbus user with which to run the tests'
+      - password:
+          name: NIMBUS_PASSWORD
+          default: ''
+          description: 'Must specified. The password of the nimbus user with which to run the tests'
+      - choice:
+          name: PARALLEL_JOBS
+          choices:
+            - '1'
+            - '2'
+            - '3'
+            - '4'
+          description: 'The parallel executor of robot tests.'
+      - string:
+          name: TEST_CASES
+          default: ''
+          description: |
+            The specific test cases to run as part of the job. These are passed as arguments to pabot so will likely be of the form "tests/manual-test-cases/Group23-Future-Tests"
+            Runs the default set of tests if left empty.
+    scm:
+      - git:
+          url: 'git@gitlab.eng.vmware.com:core-build/vic-internal.git'
+          credentials-id: 'GitLabAutomationKey'
+          branches:
+            - 'master'
+          basedir: 'vic-internal'
+      - git:
+          url: 'https://github.com/vmware/vic-tools'
+          branches:
+            - 'master'
+          basedir: 'vic-tools'
+      - git:
+          url: 'https://github.com/vmware/{repo}'
+          branches:
+            - '${{TEST_BRANCH}}'
+          basedir: '{repo}'
+          name: ''
+          refspec: '+refs/heads/*:refs/remotes/origin/* +refs/pull/*/head:refs/remotes/origin/pull/*'
+    wrappers:
+      - timeout:
+          name: absolute_timeout
+          timeout: '{build_timeout}'
+          abort: true
+          type: absolute
+      - workspace-cleanup
+    publishers:
+      - robot:
+          output-path: '{repo}/report'
+          pass-threshold: '100'
+          unstable-threshold: '100'
+      - groovy-postbuild:
+          script: |
+              def vc_build = manager.envVars['VC_BUILD']
+              def esx_build = manager.envVars['ESX_BUILD']
+              def vsphere_version = manager.envVars.get('VSPHERE_VERSION')
+              def summary = manager.createSummary("gear2.gif")
+              summary.appendText("Infrastructure Builds:<ul>", false)
+              summary.appendText("<li><b>vSphere Version</b> - ${{vsphere_version}}</li>", false)
+              summary.appendText("<li><b>vCenter</b> - ${{vc_build}}</li>", false)
+              summary.appendText("<li><b>ESXi</b> - ${{esx_build}}</li>", false)
+      - email-ext:
+          success: true
+          aborted: true
+          failure: true
+          content-type: html
+          send-to:
+            - requester
+          subject: 'vmware/{repo} PR Testing - ${{TEST_BRANCH}}'
+          body: |
```

Comment:
Could this message include more information about the test run? If you're running a few tests at the same time, it seems like figuring out which one an email is talking about might require actually opening Jenkins

Reply:
TEST_BRANCH is PR number,  what other information do you think is better to have in the email?